src/types/generated/operations.ts

```diff
@@ -55,7 +55,6 @@ export type CurrentTokenOwnershipFieldsFragment = {
     token_properties: any;
     token_standard: string;
     token_uri: string;
-    decimals: any;
```

Comment:
this is the result of running `indexer-codegen`

Reply:
This is a breaking change - have we talked with Indexer team on why they removed it?